src/shared-components/icon/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTheme } from 'emotion-theming';
+
+import Style from './style';
+
+export interface IconProps extends React.SVGProps<SVGSVGElement> {
+  className?: string;
+  fill?: string;
+  height?: string | number;
+  displayInline?: boolean;
```

Comment:
Renamed `inline` to `displayInline` to prevent an invalid DOM type error. There's only a few usages where we violate this, so it's a quick fix, but this is also technically a breaking change that I will make in the PRs that update `radiance-ui` versions.